### DISCUSSION TITLE
Domain forwarding

### DISCUSSION THREAD

**Discussion by dominikmayer** [plausible/analytics#711] [Self-Hosted Support]
My website `www.example.com` uses Plausible.

A second domain `domain.com` is forwarded to `https://www.example.com/subpage` (301 redirect).

The Plausible source list shows, on which site incoming users clicked on a link to either `domain.com` or `www.example.com/subpage`. It does not show if the link used `domain.com` or `www.example.com/subpage`.

I can add a `utm_source` parameter in the domain forwarding settings (`https://www.example.com/subpage?utm_source=domain`). But then this UTM source replaces the original interaction source (the page that the user clicked on the incoming link).

I was thinking of using only the `utm_medium` tag but the Plausible [documentation](https://plausible.io/blog/utm-tracking-tags#utm-source) says:

> The utm_source parameter is mandatory and required. Make sure to use it in all UTM tagged links.

Is there any way of tracking the domain interaction while retaining the original source?

**Comment by ukutaht** [comment]
Plausible captures all utm tags independently. I think specifying `utm_medium` or `utm_campaign` would work fine in your case. You can then get the referring site from the Top Sources list and see the number of people landing through `domain.com` in the 'medium' tab in your Top Sources.

**Reply by dominikmayer** [reply]
I tried `utm_medium` and it is shown. I will run some more tests tomorrow to see if I can click on a link and see the original source.

**Reply by dominikmayer** [reply]
I did some more tests (which is not that easy, as Plausible only tracks the source of the first visit, I think).

It seems that the parameter is only retained if I enter `www.domain.com` in the browser bar. It resolves to `https://www.example.com/subpage?utm_medium=domain.com`.

When I click on a link, on the other hand, it seems that the parameter is stripped away ([Dillinger](http://dillinger.io)) or replaced with the parameters of the linking site (Facebook: `https://www.example.com/subpage?fbclid=…`).
